Dockerfile

```diff
@@ -1,13 +1,19 @@
 FROM golang:alpine
 MAINTAINER Jimmy Zelinskie <jimmyzelinskie@gmail.com>
 
-RUN apk update && apk add curl git
-RUN curl https://glide.sh/get | sh
+# Install OS-level dependencies
+RUN apk update && \
+    apk add curl git && \
+    curl https://glide.sh/get | sh
 
+# Copy our source code into the container
 WORKDIR /go/src/github.com/chihaya/chihaya
 ADD . /go/src/github.com/chihaya/chihaya
+
+# Install our golang dependencies and compile our binary.
```

Comment:
This is the only comment ending in a period. We must have them all end the same way.